routes/api/v1/foods.js

```diff
@@ -16,4 +16,24 @@ router.get("/", function (req, res) {
     });
 });
 
+router.delete('/:id', function(req, res, next) {
+  Food.destroy({
+    where: {
+      id: req.params.id
+    }
+  })
+	.then(food => {
+     if (food !== 1)
+	{throw new Error("Not Found")}
+	})
+  .then(food => {
+    res.setHeader('Content-Type', 'application/json');
+    res.status(204).send(JSON.stringify(food));
```

Comment:
With a status code 204, you don't need to send anything back.

Reply:
Cool.  I will remove it